This test, on three occasions, is failing with the error:

  EOF: End Of File (EOF) in read_nonblocking(). Braindead platform.

--- duplicity-0.6.24/testing/functional/test_final.py	2014-09-28 13:14:52.146001614 -0500
+++ duplicity-0.6.24/testing/functional/test_final.py	2014-09-28 13:13:20.333546342 -0500
@@ -156,13 +156,6 @@
         self.run_duplicity(options=["remove-older-than", "50000", "--force", self.backend_url])
         self.assertEqual(self.get_backend_files(), second_chain)
 
-    def test_piped_password(self):
-        """Make sure that prompting for a password works"""
-        self.set_environ("PASSPHRASE", None)
-        self.backup("full", "testfiles/empty_dir",
-                    passphrase_input=[self.sign_passphrase, self.sign_passphrase])
-        self.restore(passphrase_input=[self.sign_passphrase])
-
 
 class OldFilenamesFinalTest(FinalTest):
 
